submissions/f3d0t/js-oop/app.js

```diff
@@ -116,17 +116,11 @@ Player.prototype.handleInput = function (keycode) {
 	this.updatePosition(newX, newY);
 };
 
-const enemyStartPostionsY = [];
-for (let i = 1; i <= 3; i++) {
-	enemyStartPostionsY.push(cellInfo.height * i - cellInfo.beingPositionCorrection);
-}
-enemyStartPostionsY.forEach((posY) => {
-	let enemy = new Enemy(-enemyInfo.width, posY);
-	allEnemies.push(enemy);
-});
-
 const player = new Player(playerInfo.startPositionX, playerInfo.startPositionY);
 
+const enemyStartPostionsY = [1, 2, 3].map((rowNumber) => cellInfo.height * rowNumber - cellInfo.beingPositionCorrection);
```

Comment:
Please made creation more automatically. What you will do if you will need 100500 bugs. 

`Array.from({ length: })` can help you i guess

Reply:
This way is really cool